docs/getting-started.md

```diff
@@ -9,13 +9,13 @@ Spicetify is a multiplatform command-line tool to customize the official Spotify
 
 ### Windows
 
-This is the installation method what we recommend for most users. It is the fastest and most reliable way to install Spicetify. It also includes the [**Spicetify Marketplace**](https://github.com/spicetify/spicetify-marketplace) that gives you access to a tab in Spotify's sidebar that allows you to search for and install _themes_, _extensions_ and _snippets_.
+This is the installation method what we recommend for most users. It is the fastest and most reliable way to install Spicetify. It also includes the [**Spicetify Marketplace**](https://github.com/spicetify/spicetify-marketplace) that gives you access to a tab in Spotify's sidebar that allows you to search for and install _themes_, _extensions_, and _snippets_.
```

Comment:
Do we want to change the **Spicetify Marketplace** to just marketplace?

Reply:
I think it's _the_ Spicetify Marketplace, so that's fine.